docs/setup/install-with-keadm.md

```diff
@@ -218,213 +218,16 @@ MQTT is installed in this host
 KubeEdge edgecore is running, For logs visit: journalctl -u edgecore.service -xe
 ```
 
-### Deploy demo on edge nodes
+## Deploy demo on edge nodes
```

Comment:
i believe that this indent is wrong since `application demo` should not included under `keadm join` section.